searchIndex-platform/module/search-api/search-manager/app/managers/BasePlaySearchManager.java

```diff
@@ -266,4 +270,37 @@ protected String getMessage(Response res) {
 		}
 		
 	}
+
+	/**
+	 * @param context
+	 * @param filters
+	 */
+	private void populateObjectField(Map<String, Object> context, Map<String, Object> filters) {
```

Comment:
can you please rephrase the method name, as this is specific to Dialcodes

Reply:
changed to populateTarget
DialObject